README.rst

```diff
@@ -116,15 +120,15 @@ To build the PHP driver for Windows:
 
    - :code:`<arch>` is your CPU architecture (:code:`x64` or :code:`x86`).
    - :code:`<build>` is the type of binary that you want to build (:code:`Release` or :code:`Debug`).
-   - :code:`<visual studio version>` is the version of Visual Studio that you are using (:code:`VS14` or :code:`VS15`).
+   - :code:`<visual studio version>` is the version of Visual Studio that you are using (:code:`VS14`, :code:`VS15`, or :code:`VS16`).
    - :code:`<path to PHP SDK>` is the path to the directory where the PHP SDK should be downloaded.
      **Do not create this directory.** The script creates this directory for you when downloading the PHP SDK.
 
    For example:
 
    .. parsed-literal::
 
-       .\\scripts\\setup_php_sdk.bat x64 Release |win-vs-version| C:\\php-sdk
+       .\\scripts\\setup_php_sdk.bat x64 Release VS16 C:\\php-sdk
 
 #. Download and install the PHP binaries, or build PHP yourself.
 
```

Comment:
This step is always needed even when downloading PHP binaries directly. To build the driver PHP source code is needed.
So maybe change this step like "download and build PHP source code", and make it mandatory step, not optional.